stackdriver.go

```diff
@@ -125,7 +128,20 @@ func (e *Exporter) ExportView(vd *view.Data) {
 
 // ExportSpan exports a SpanData to Stackdriver Trace.
 func (e *Exporter) ExportSpan(sd *trace.SpanData) {
-	e.traceExporter.ExportSpan(sd)
+	newSD := e.sdWithDefaultTraceAttributes(sd)
+	e.traceExporter.ExportSpan(newSD)
+}
+
+func (e *Exporter) sdWithDefaultTraceAttributes(sd *trace.SpanData) *trace.SpanData {
+	newSD := *sd
```

Comment:
I'm concerned about the additional allocation here. Can we make this a no-op if o.DefaultTraceAttributes is empty? Either that would you consider adding the DefaultTraceAttributes directly to the exported protobufs in copyAttributes (trace_proto.go:166)?

Reply:
I went with making it a no-op because for doing it in `copyAttributes` I would need to hand the options down the stack to that function. Also this would circumvent the bundler size calculation AFAIU.